ci/matrix.yaml

```diff
@@ -54,7 +60,12 @@ workflows:
     - {jobs: ['build'], project: 'cudax', ctk: ['curr'], std: 'all', cxx: ['gcc', 'clang'], cpu: 'arm64'}
     - {jobs: ['test'],  project: 'cudax', ctk: ['curr'], std: 20,    cxx: ['gcc', 'clang', 'msvc'], gpu: 'rtx2080'}
     # Python and c/parallel jobs:
-    - {jobs: ['test'], project: ['cccl_c_parallel', 'python'], gpu: 'rtx2080'}
+    - {jobs: ['build_cuda_cccl', 'build_cuda_cooperative', 'build_cuda_parallel'], project: 'python', py_version: '3.10'}
+    - {jobs: ['build_cuda_cccl', 'build_cuda_cooperative', 'build_cuda_parallel'], project: 'python', py_version: '3.12'}
+    - {jobs: ['build_cuda_cccl', 'build_cuda_cooperative', 'build_cuda_parallel'], project: 'python', py_version: '3.13'}
+    - {jobs: ['test_cuda_cccl', 'test_cuda_cooperative', 'test_cuda_parallel'], project: 'python', gpu: 'rtxa6000', py_version: '3.10'}
+    - {jobs: ['test_cuda_cccl', 'test_cuda_cooperative', 'test_cuda_parallel'], project: 'python', gpu: 'rtxa6000', py_version: '3.12'}
+    - {jobs: ['test_cuda_cccl', 'test_cuda_cooperative', 'test_cuda_parallel'], project: 'python', gpu: 'rtxa6000', py_version: '3.13'}
```

Comment:
Test jobs cannot run on consumer cards until they support the latest driver (or at least CUDA 12.8 driver).

Forward compatibility does not work on consumer cards, and the `cuda_parallel` wheels are built against CUDA 12.8.

Reply:
Soon to be 12.9: https://github.com/NVIDIA/cccl/pull/4624

Reply:
Added a note